dart/lib/src/protocol/sentry_stack_trace.dart

```diff
@@ -5,11 +5,11 @@ import 'sentry_stack_frame.dart';
 /// Stacktrace holds information about the frames of the stack.
 @immutable
 class SentryStackTrace {
-  const SentryStackTrace({
-    @required List<SentryStackFrame> frames,
-    Map<String, String> registers,
+  SentryStackTrace({
+    required List<SentryStackFrame> frames,
+    Map<String, String>? registers,
   })  : _frames = frames,
-        _registers = registers;
+        _registers = Map.from(registers ?? {});
```

Comment:
Check if I can use `_registers = Map.from(registers ?? const {});`
Also probably enable the following lints in another PR:

- https://dart-lang.github.io/linter/lints/prefer_const_constructors.html
- https://dart-lang.github.io/linter/lints/prefer_const_declarations.html
- https://dart-lang.github.io/linter/lints/prefer_const_literals_to_create_immutables.html

Const ist good because https://stackoverflow.com/a/50431087/12002462

Reply:
why don't we use `const` here?